ui/App/views/UserManagement/components/CreateUserForm.jsx

```diff
@@ -25,7 +25,9 @@ const CreateUserForm = ({updateUserList}) => {
                        ref={register({required: true})}
                        id="username"
                        name="username"
-                       type="text" placeholder="Username"/>
+                       type="text"
```

Comment:
We should use the components for inputs and labels. I guess this form was created before the components.
If you like you can refactor it and use the components.